change/@ni-nimble-blazor-1cbfc49b-865a-4368-ab38-15ee937d0420.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Oh, so the only changes that should have a type are the ones that have impact on the user, I understand, I put 'none' there

Reply:
Yeah, it's all about tracking what bits are included in the built npm package. Since we don't ship our tests, the build output isn't changed so we can mark it "none" and the beachball publishing tool won't even bother releasing a new version.

(for this PR it's actually irrelevant because the nimble-components change will trigger a new NimbleBlazor package anyway but still good to be correct)